package.json

```diff
@@ -1,45 +1,45 @@
 {
-    "name": "jasmine-node-helpers",
-    "version": "1.0.0",
-    "description": "Public and free software for helping test NodeJS projects using Jasmine-Node.",
-    "main": "/index.js",
-    "scripts": {
-        "test": "istanbul cover jasmine-node --captureExceptions spec/ && codecov && eslint .",
-        "watch": "jasmine-node lib spec/ --autotest --watch"
+  "name": "jasmine-node-helpers",
+  "version": "1.0.0",
+  "description": "Public and free software for helping test NodeJS projects using Jasmine-Node.",
+  "main": "/index.js",
+  "scripts": {
+    "test": "istanbul cover jasmine --captureExceptions && codecov",
+    "watch": "jasmine-node lib spec/ --autotest --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tests-always-included/jasmine-test-helpers.git"
+  },
+  "keywords": [
+    "test",
+    "helper",
+    "node"
```

Comment:
Add some useful keywords like "jasmine", "promise", etc.  Jasmine should probably even be first.